src/components/EventCard/EventCard.js

```diff
@@ -25,25 +25,25 @@ const EventCard = ({ event, className }) => {
   // TODO: Seems like the featured_image_url is no longer coming back -- follow up when services more stable
 
   return (
-    <li className={`${styles.card} ${className || ''}`}>
-      <div className={styles.imageWrapper}>
-        <img
-          src={featuredImageUrl || 'http://www.fillmurray.com/550/450'}
-          alt="featured event"
-        />
-      </div>
-      <div className={styles.contentWrapper}>
-        <div className={styles.dateLocation}>
-          <DateBlock date={startDate} />
-          { renderLocation(loc) }
+    <Link to={`/event/${_id}`}>
```

Comment:
You can't have an `li` inside an `a` tag.  Just change the nesting and `ul li a { display:block; }`